script/v2/V2DeployContracts.s.sol

```diff
@@ -21,72 +22,81 @@ contract V2DeployContracts is Script, HelperV2 {
 
     function run() public {
         ConfigAddressesV2 memory addresses = getConfigAddresses(false);
-        // address policy = addresses.policy;
         address weth = addresses.weth;
         address treasury = addresses.treasury;
         address emissionToken = addresses.y2k;
         address policy = addresses.policy;
 
-        console2.log("Address admin", addresses.admin);
-        console2.log("Address arbitrum_sequencer", addresses.arbitrum_sequencer);
-        console2.log("Address gelatoOpsV2", addresses.gelatoOpsV2);
-        console2.log("Address gelatoTaskTreasury", addresses.gelatoTaskTreasury);
+        // console2.log("Address admin", addresses.admin);
```

Comment:
@0xnyxos i we should prob uncomment before we merge this to main.

Reply:
Just to clarif, we want to uncomment the console logs from admin --> gelatoTreasury?